joomla/joomla-cms
# [3.10] backport #26293 into 3.10-dev thanks @ChristineWk

Pull Request for Issue # .

### Summary of Changes

[3.10] backport #26293 into 3.10-dev thanks @ChristineWk

### Testing Instructions

- install 3.10-dev
- install patchtester
- go to the database and clear the field `manifest_cache` for the patchtester record in the extensions table.
- set the [nightly major update server](https://update.joomla.org/core/nightlies/next_major_list.xml)
- notice the pre update checker
- notice that the pre update checker shows nothing as version for the patchtester and raises a notice
- apply this patch
- the updater now shows the string `Unknown version in this case and should not raise and notice any more.

### Actual result BEFORE applying this Pull Request

Notice: Trying to get property 'version' of non-object in /home/xxx/xx/xxx/xxx/xxx/administrator/components/com_joomlaupdate/models/default.php on line 1410

### Expected result AFTER applying this Pull Request

`Unknown version`

### Documentation Changes Required

none
